src/index/IndexImpl.h

```diff
@@ -332,6 +332,11 @@ class IndexImpl {
 
   Index::WordEntityPostings getEntityPostingsForTerm(const string& term) const;
 
+  Index::WordEntityPostings getUnadjustedEntityPostingsForTerm(
+      const string& term) const;
+
+  size_t getIndexOfBestSuitedElTerm(const vector<string>& terms) const;
+
```

Comment:
Somewhere you should comment these functions.
Also the name of the `getUnadjusted..` function is rather confusing.
`getEntityMentionsForWord` is more fitting. And also it should be documented that this
doesn't filter exactly but gives a superset of the words.